operations/finance.md

```diff
@@ -416,29 +416,59 @@ Returns all accounting items of the enterprise that were consumed \(posted\) or
 | `SettlementId` | string | optional | Identifier of the settlement. |
 | `SettledUtc` | string | optional | Settlement date and time in UTC timezone in ISO 8601 format. |
 
-## Get all bills by ids
+## Get all bills
 
-Returns all bills with the specified ids.
+Returns all bills, possible filtered by customers, identifiers and other filters.
 
 ### Request
 
-`[PlatformAddress]/api/connector/v1/bills/getAllByIds`
+`[PlatformAddress]/api/connector/v1/bills/getAll`
 
 ```javascript
 {
     "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
     "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
     "BillIds": [
         "e654f217-d1b5-46be-a820-e93ba568dfac"
-    ]
+    ],
+    "CustomerIds": [
+        "fe795f96-0b64-445b-89ed-c032563f2bac"
+    ],
+    "TimeFilter": "Created",
+    "States": [
```

Comment:
@vitasamek This is bad example, because you're getting all bills but still explicitly specify filter so the DB will have unnecessary WHERE. Consider changing this to just State which is nullable. null means all states or you specify exact state you want.